playground/nuxt.config.ts

```diff
@@ -8,7 +8,13 @@ export default defineNuxtConfig({
   algolia: {
     apiKey: process.env.ALGOLIA_SEARCH_API_KEY ?? 'apiKey',
     applicationId: process.env.ALGOLIA_APPLICATION_ID ?? 'applicationId',
-    lite: false, // by default set to 'true',
+    lite: false, // by default set to 'true'
+    docSearch: {
+      apiKey: process.env.ALGOLIA_DOCSEARCH_API_KEY ?? 'apiKey',
+      applicationId: process.env.ALGOLIA_DOCSEARCH_APPLICATION_ID ?? 'applicationId',
+      indexName: process.env.ALGOLIA_DOCSEARCH_INDEX_NAME ?? 'indexName',
+      facetFilters: process.env.ALGOLIA_DOCSEARCH_FACET_FILTERS ? process.env.ALGOLIA_DOCSEARCH_FACET_FILTERS.split(',') : []
```

Comment:
Not sure that the indexName and facetFilters should be part of the global nuxt config.ts. As you may have several situations where you would like to use docsearch (i.e. one global search and one search as a part of the search page - not a common case but could happen). 

So I would prefer to pass these variables directly into the component module as it was before